CMakeLists.txt

```diff
@@ -646,6 +640,12 @@ if(WINCE)
 	set(HAVE_PRAGMA_PACK 1)
 	set_with_reason(vehicle/file "wince: currently broken" FALSE)
 	set_with_reason(vehicle/wince "wince detected" TRUE)
+	set_with_reason(support/ezxml "Windows detected" TRUE)
+	set_with_reason(speech/espeak "Windows detected" TRUE)
+	#speech/espeak does use windows threads, and therefore needs only parts of
+	#espeak lib. INTERNAL_ESPEAK_COMPLETE controls this.
+	set_with_reason(INTERNAL_ESPEAK_COMPLETE "Windows detected" FALSE)
+	set_with_reason(support/espeak "Windows detected" TRUE)
```

Comment:
Hmm, you're moving the enablement of these (support) packages to WINCE only. Sure they are not required by WIN32 builds as they have been before?

Reply:
I think that it's because of a discussion we've had on IRC.
espeak isn't great, and @hoehnp was testing using the native windows TTS api.

Reply:
OK. this then disables espeak for win32 for now until @hoehnp finishes his work. Never mind.

Reply:
:+1: